build.assets/robotest/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG BASE
```

Comment:
A comment & link to the relevant bug wouldn't go awry in this file.

Reply:
This entire file has been replaced by a simpler `--user=$(id -u):$(id -g)` in 
2e7bfe1